src/main/scala/advancepassengerinfo/importer/DqApiFeed.scala

```diff
@@ -24,14 +26,15 @@ case class DqApiFeedImpl(fileNamesProvider: FileNames,
   override def processFilesAfter(lastFileName: String): Source[String, NotUsed] =
     Source
       .unfoldAsync((lastFileName, List[String]())) { case (lastFileName, lastFiles) =>
-        markerAndNextFileNames(lastFileName).map {
-          case (nextFetch, newFiles) => Option((nextFetch, newFiles), (lastFileName, lastFiles))
+        val fallbackFileName = "drt_dq_" + yyyyMMdd(SDate.now().addDays(-1)) + "_000000_0000.zip"
```

Comment:
This is an interesting approach mate, but it's not quite right. If you make up a non-existent file name from 2 days ago then 2 things will happen:
1. We'll have another error when it fails to download the file since it doesn't exist
2. It'll then try to fetch all files dated since that one, which means we'll reprocess 2 days of files that we have already processed

Reply:
Yeah , Agree .  I thought commit and discuss would be better .  Moving to empty list .